.github/workflows/static-analysis.yml

```diff
@@ -68,4 +68,27 @@ jobs:
             -Dsonar.python.coverage.reportPaths=coverage/coverage.xml
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
\ No newline at end of file
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+
+  trufflehog-scan:
+    runs-on: ubuntu-latest
```

Comment:
Can we give this job a nicely formatted name please, just so it is clear what Trufflehog does, and looks nice during the pipeline.

E.g.
```
trufflehog-scan:
  name: Secret scan using Trufflehog
```